x-pack/plugins/ml/public/file_datavisualizer/components/fields_stats/fields_stats.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React, {
+  Component,
+} from 'react';
+
+import { FieldStatsCard } from './field_stats_card';
+
+export class FieldsStats extends Component {
+  constructor(props) {
+    super(props);
+
+    this.fields = createFields(this.props.results);
+    console.log(this.fields);
```

Comment:
Did you mean to leave this `console.log` in?

Reply:
yes, i'm leaving logging in as it's a work in progress